tests/profile_service/profile.spec.js

```diff
@@ -63,16 +63,25 @@ const createAttribute = (name, value) => new Attribute({
   verifiers: [],
 });
 
+const profileDataAsObject = loadProfileData();
+const profileDataAsArrayWithSameAttributes = [...loadProfileDataArray(), ...loadProfileDataArray()]
+  .map((attr, index) => {
+    if (['selfie', 'document_images'].includes(attr.name)) {
+      return Object.assign({}, attr, { id: `${attr.name}-${index}` });
```

Comment:
Why not use destructuring?

Reply:
This is all over the project, this is not using it due to the Node v6 support.

Once we move to 12 minimum, we'll be chasing the code to remove Object.assign.